app/models/user.server.ts

```diff
@@ -13,7 +13,7 @@ export interface User extends PrismaUser {
    username, admins (identified by group) will start with an empty string
    that can be changed to another users username on the admin portal for 
    them to impersonate */
-  effectiveUsername: string;
+  effectiveUsername?: string;
```

Comment:
The type here should be `User['username']` so it matches what `username` is when we pass it around.